docs/api/actions-and-connectors/execute.asciidoc

```diff
@@ -50,6 +50,104 @@ depending on the connector type. For information about the parameter properties,
 refer to <<action-types>>.
 +
 --
+.Jira connectors
+[%collapsible%open]
+====
+`subAction`::
+(Required, string) The action to test. Valid values include: `fieldsByIssueType`,
+`getFields`, `getIncident`, `issue`, `issues`, `issueTypes`, and `pushToService`.
+
+`subActionParams`::
+(Required^*^, object) The set of configuration properties, which vary depending
+on the `subAction` value. This object is not required when `subAction` is
```

Comment:
The object is always required. When the `subAction` is getFields` or `issueTypes` then the `subActionParams` should be an empty object (`{}`)

Reply:
I was able to run them successfully in 8.4.0 without the empty object. For example:

> curl -X POST "https://localhost:5601/api/actions/connector/b3aad810-edbe-11ec-82d1-11348ecbf4a6/_execute?pretty" -H 'kbn-xsrf: true' -H 'Content-Type: application/json' -d '
{"params":{"subAction":"issueTypes"}}' --cacert ca.crt -u elastic

Is this supported but not recommended?

Reply:
I was not aware that you can omit the `thesubActionParams` if it is defined as an empty object (no params basically) on the schema. No, it is fine. TIL, thanks!